client/cody/src/chat/protocol.ts

```diff
@@ -57,9 +55,23 @@ export interface AuthStatus {
     authenticated: boolean
     hasVerifiedEmail: boolean
     requiresVerifiedEmail: boolean
+    siteHasCodyEnabled: boolean
+    siteVersion: string
+}
+
+export const authStatusInit = {
```

Comment:
```suggestion
export const defaultAuthStatus = {
```

I think that's clearer.